src/ContaoCommunityAlliance/DcGeneral/DataDefinition/Definition/Properties/DefaultCompoundProperty.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * This file is part of contao-community-alliance/dc-general.
+ *
+ * (c) 2013-2017 Contao Community Alliance.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    contao-community-alliance/dc-general
+ * @author     Sven Baumann <baumann.sv@gmail.com>
+ * @copyright  2013-2017 Contao Community Alliance.
+ * @license    https://github.com/contao-community-alliance/dc-general/blob/master/LICENSE LGPL-3.0
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\Properties;
+
+use ContaoCommunityAlliance\DcGeneral\DataDefinition\Definition\Properties\Collection\SubPropertyInterface;
+use ContaoCommunityAlliance\DcGeneral\Exception\DcGeneralInvalidArgumentException;
+
+/**
+ * Default implementation of a property with collection definition.
```

Comment:
phpDoc has to be adapted to the name `SubProperty`.

Reply:
change